src/M6Web/Component/RedisMock/RedisMockFactory.php

```diff
@@ -314,6 +314,9 @@ protected function getMethodSignature(\ReflectionMethod $method)
             }
             // paramName
             $signature .= '$' . $parameter->getName();
+
+            $variadic = PHP_VERSION_ID >= 50600 ? $parameter->isVariadic() : false;
```

Comment:
We officially dropped support to versions prior to `7.1`
https://github.com/M6Web/RedisMock/blob/master/composer.json#L18

I think you can safely remove this check. Furthermore, I don't see how you could mock a variadic function if your Php version does not support this feature.